### PR TITLE
Add review-violet-evergarden

### PR DESCRIPTION
**Pull request by rayriffy**
Waiting for QC

### CODE REVIEW

**Comment by rayriffy** [comment]
I will open another PR for typo fix

File: src/pages/review-violet-evergarden/index.md [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+
+## Trivia
+
+ก่อนหน้านี้เป็น Light Novel ต่อมาก็ได้เข้าไปประกวดในงาน 5th Kyoto Animation Awards และได้รางวัลชนะเลิศกลับมา แล้วได้สานงานต่อให้เป็นอนิเมชั่นโดย Kyoto Animation และวิดิโอที่ได้ดูด้านบนนั่นก็คือ PV ตัวแรกที่ปล่อยออกมาในปี 2016 ซึ่งตอนนั้นก็ได้แต่อ้าปากค้างตามๆ กันไป ด้วยลายเส้นและรายละเอียดที่เห็นออกมาสามารถบอกได้เลยว่าค่ายอนิเมชั่นนี้ใส่ใจในคุณภาพงานขนาดไหน
+
+## เนื้อเรื่องย่อ
+
+![Episode 2](./dldcQ3wjyU.jpg)
+
+หลังจากสงครามของประเทศ Leidenschaftlich ได้สิ้นสุดลง **Violet Evergarden** ได้เดินทางไปเริ่มต้นชีวิตใหม่หลังฟื้นตัวอยู่ที่โรงพยาบาล โดยทำงานเป็น Auto Memories Doll ที่ CH Postal Company ของ **Hodgins** ซึ่ง Hodgins เป็นเพื่อนสนิทของ **Gilbert** ผู้ที่คอยดูแล Violet มาตั้งแต่เด็ก โดยมีหน้าที่หลักคือ**เขียนจดหมายให้ลูกค้าที่ไม่สามารถเขียนจดหมายด้วยตัวเอง หรือสื่อความหมายออกมาได้** แล้ว Violet ก็ได้เดินทางไปตามสถานที่ต่างๆ เพื่อรับใช้ลูกค้า และค้นหาความหมายของคำว่า *"ฉันรักเธอ"* ซึ่งเป็นคำพูดสุดท้ายของ Gilbert ก่อนที่ Violet จะฟื้นขึ้นมาอีกทีที่โรงพยาบาล แล้วเมื่อเวลาผ่านไปเรื่อยๆ ความลับต่างๆ ที่อยู่ในใจของ Violet ก็ได้ถูกเปิดเผยออกมา
```
Comment (rayriffy) [inline src/pages/review-violet-evergarden/index.md]: OK

File: src/pages/review-violet-evergarden/index.md [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+
+แล้วต่อมา Violet ก็ได้เดินทางไปเมือง Drossel มาพบกับ **Princess Charlotte** เพื่อที่จะเขียนจดหมายรักให้กับ **Prince Damian** แห่งเมือง Fluegel ที่ถูกพรากกันตั้งแต่ได้รู้จักกับ 'รักแรกพบ' เนื่องจากสงครามที่เกิดขึ้น ซึ่งนี่ก็เป็นนึงในตอนที่เราชอบนะ ได้ไม่ใช่ที่สุด
+
+อันนี้ก็เป็นเพียงตัวอย่างให้ดูว่าเป็นยังไงในแต่ละตอน โดยตามที่บอกเลยว่า Violet จะได้รับรู้ในความเป็นมนุษย์มากขึ้นจากการพบปะกับผู้คนเรื่อยๆ
+
+## ประวัติเก่าอันโชกโชน
+
+![Episode 8](./d3G6ZJiY75.jpg)
+
+เมื่อ Violet ยังเป็นเด็กๆอยู่ Gilbert ได้รับเลี้ยงเธอเอาไว้ แต่นี่หน้าตกใจคือเธอถูกฝึกมาเพื่อเป็นเครื่องมือทางสงครามในช่วงที่ประเทศกำลังมีสงครามกันอยู่ ซึ่งก็ดีที่ได้เห็นเธอพยายามปรับตัวเรื่อยๆในช่วงที่เธอทำงานเป็น Auto Memories Doll
```
Comment (rayriffy) [inline src/pages/review-violet-evergarden/index.md]: Not OK

File: src/pages/review-violet-evergarden/index.md [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+
+ปกติเวลาเราดู Anime แล้วมี OP เปิดขึ้นมา เราก็แบบว่า ดูแค่ 4-5 ตอนแรกๆ แล้วก็ Skip แต่เรื่องนี้เป็นเรื่องแรกในรอบ 3 ปีนี้ที่เราไม่กด Skip OP เลยแม้แต่ตอนเดียว คือแบบ เพลงดี ([TRUE - Sincerely](https://youtu.be/uwph0dv9E6U)) เดี๋ยวเราจะแนบเพลงไว้ให้ฟังด้านล่าง ภาพดี คือมันดีไปหมดทุกด้านอ่ะ
+
+`youtube:uwph0dv9E6U`
+
+## เรื่องราวของเด็กสาวคนหนึ่งที่จะทำให้คุณจมไปกับอารมณ์ของตัวละคร
+
+![Episode 2](./lGycZhpoNs.jpg)
+
+ในแต่ละตอน Violet จะเดินทางพบกับผู้คนใหม่ๆ เพื่อที่จะได้เรียนรู้ถึงอารมณ์ ความรู้สึก และความเป็นมนุษย์มากขึ้นผ่านจากพวกเขา และเข้าใจถึงหัวอกหัวใจของผู้คนเหล่านั้น เพราะตั้งแต่ตอนที่ Violet เป็นเด็ก เธอได้รับการเลี้ยงดูโดย Gilbert ซึ่งทำให้เธอ**ใช้ชีวิตแบบทำตามคำสั่งมาตลอด** จะสังเกตเห็นได้ชัดเจนมากๆ ในช่วง 3 ตอนแรก
```
Comment (KrKaster) [inline src/pages/review-violet-evergarden/index.md]: OK

**Pull request merged**